Makefile.dep

```diff
@@ -87,6 +87,11 @@ ifneq (,$(filter ng_netbase,$(USEMODULE)))
   USEMODULE += ng_pktbuf
 endif
 
+ifneq (,$(filter ng_udp,$(USEMODULE)))
+  USEMODULE += ng_netbase
+  USEMODULE += ng_inet_csum
+endif
+
```

Comment:
This needs to be moved above l83, otherwise `ng_netbase`'s dependencies won't be included.